### PR TITLE
Allocate size for jaeger log args

### PR DESCRIPTION
**Pull request by madhuravi**

### CODE REVIEW

**Comment by madhuravi** [comment]
@alsamylkin

File: tracing/tracer.go [hunk 1]
```diff
@@ -79,11 +79,12 @@ func (jl *jaegerLogger) Error(msg string) {
 func (jl *jaegerLogger) Infof(msg string, args ...interface{}) {
 	// TODO (madhu) Zap logger requires even number of arguments, add a key to each arg
 	// This is a hack in place till jaeger directly takes a zap logger
-	var logArgs []interface{}
+	targetLen := len(args) * 2
+	var targetArgs = make([]interface{}, targetLen)
```
Comment (alsamylkin) [inline tracing/tracer.go]: I think it should be 
`make([]interface{}, 0, targetLen)`

Comment (sectioneight) [inline tracing/tracer.go]: no need var `var`, you can do `targetArgs := make(...)`

Comment (madhuravi) [inline tracing/tracer.go]: Alex, if capacity is not specified, it defaults to len. Just specifying length should do.

Comment (alsamylkin) [inline tracing/tracer.go]: I think you are appending to the end of the slice, so targetArgs would have len == 2*targetLen ;)

Comment (madhuravi) [inline tracing/tracer.go]: oops you're right.

**Pull request merged**